### PR TITLE
[DON'T FORWARD] LPS-135323 Reverting language targeting when creating an experience

### PR DESCRIPTION
**Pull request by sandrodw3**
Hi @liferay-echo! 😄

We are reverting this ticket: https://issues.liferay.com/browse/LPS-124170.

That ticket introduced the possibility to choose some languages when creating an experience, so then, when switching to that experience, only those languages were available in the language selector.

With this revert, it should be working like it was before the story (users don't select any language when creating an experience and all of them are available when switching to it).

I have tested experience creation/modification/removal, and also switching to the experience and adding some fragments works well.

I'm sending this to Echo because it affects the language selector. Could you plase give it a try just in case I forgot something? 🙂

If you think everything seems to work fine, please tell us, but don't forward, as @4lejandrito wants to adjust the commits messages before sending. 

Thanks a lot!

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ export default function getLanguages(
 	if (
 		!availableSegmentsExperiences ||
 		!segmentsExperienceId ||
-		!availableSegmentsExperiences[segmentsExperienceId] ||
 		segmentsExperienceId === config.defaultSegmentsExperienceId
 	) {
 		return availableLanguages;
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js:27]: I think we need to remove this file completely, right? There are usages in CurrentLanguageFlag that needs update also :)

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js:27]: Yep, I missed that one, thanks! I'll update it right now 🙂

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js]: Done! (https://github.com/liferay-echo/liferay-portal/pull/5151/commits/fc22871695e2eff53bec209eafde7a18450b6c3c#diff-e2a0423e5b7bd225047150d252839a35ba5e80a1de568257e4ff22b11c2a5bd4R19)

File: modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateSegmentsExperienceMVCActionCommand.java [hunk 2]
```diff
@@ -58,21 +55,12 @@ protected JSONObject doTransactionalCommand(
...
-		typeSettingsUnicodeProperties.setProperty(
-			PropsKeys.LOCALES, StringUtil.merge(languageIds));
 
 		return JSONUtil.put(
 			"segmentsExperience",
 			_segmentsExperienceService.updateSegmentsExperience(
 				segmentsExperienceId, segmentsEntryId,
-				Collections.singletonMap(LocaleUtil.getSiteDefault(), name),
-				true, typeSettingsUnicodeProperties));
+				Collections.singletonMap(LocaleUtil.getDefault(), name), true));
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/UpdateSegmentsExperienceMVCActionCommand.java:63]: @4lejandrito Have you tried to reproduce this issue after the changes: https://issues.liferay.com/browse/LPS-128205

We may need to preserve `Collections.singletonMap(LocaleUtil.getSiteDefault()`

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/utils/getLanguages.js]: Nice :)

**Comment by victorg1991** [comment]
Apart for the comment about the `Collections.singletonMap(LocaleUtil.getSiteDefault())` I think this is good to go :) @sandrodw3 @4lejandrito

**Comment by sandrodw3** [comment]
> Apart for the comment about the `Collections.singletonMap(LocaleUtil.getSiteDefault())` I think this is good to go :) @sandrodw3 @4lejandrito

Thanks a lot @victorg1991 🥰

**Comment by 4lejandrito** [comment]
ci:close

**Pull request closed without merging**